processFlows.json

```diff
@@ -1705,7 +1705,27 @@
           }
         }
       },
+
       { "op": "And" },
+      {
+        "restriction": {
+          "SenderHasOutputRole": {
+            "index": 0,
+            "roleKey": "MemberA"
+          }
+        }
+      },
+      {
+        "restriction": {
+          "SenderHasOutputRole": {
+            "index": 0,
+            "roleKey": "MemberB"
+          }
+        }
+      },
+      { "op": "Or" },
+      { "op": "And" } ,
+
```

Comment:
So these roles don't exist on demand tokens and I don't think this is logically necessary. We guarentee that the `match2` is attached to these tokens. In an earlier flow we make guarentees about the owners of those to the members on the match.

In otherwords I think these checks are unnecessary.

Reply:
OK, I'll remove them